### PR TITLE
Bug 1215906 - Update home screen font style to match spec. r=gmarty

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=12f29f18bfb522bf1b89b006e7b3794c22e335f5

File: apps/homescreen/style/apps-panel.css [hunk 1]
```diff
@@ -31,9 +31,18 @@ gaia-app-icon {
...
   color: white;
-  text-shadow: rgba(0, 0, 0, 0.6) 0 0.05rem 0.25rem;
+  text-shadow: rgba(0, 0, 0, 0.5) 0 0.1rem 0.3rem;
+}
+
+/* Small icons are scaled down 0.75, so apply the reverse transforms to the
+ * font so that it conforms to spec.
+ */
+#apps.small gaia-app-icon {
+  font-size: 1.6rem;
```
Comment (gmarty) [inline apps/homescreen/style/apps-panel.css]: Maybe `font-size: cacl(1.2rem * 0.75);` here to make conversion more explicit?

Comment (Cwiiis) [inline apps/homescreen/style/apps-panel.css]: Good idea... I wonder if the same can be done in the text-shadow declaration... I'll try it out.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b25a0b0ae94e0f078d207a992239f01773baba8b

**Pull request merged**